src/types.ts

```diff
@@ -385,12 +389,33 @@ export interface OpenSeaAsset extends Asset {
   transferFeePaymentToken: OpenSeaFungibleToken | null
 }
 
+export interface LastSale {
+  eventType: string
+  auctionType: string
+  totalPrice: string
+  transaction: Transaction
+}
+
+export interface Transaction {
+  id: number
+  fromAccount: OpenSeaAccount
+  toAccount: OpenSeaAccount
+  createdDate: string
+  modifiedDate: string
+  transactionHash: string
+  transactionIndex: string
+  blockNumber: string
+  blockHash: string
+  timestamp: string
+}
+
 /**
  * Full annotated Fungible Token spec with OpenSea metadata
  */
 export interface OpenSeaFungibleToken extends Token {
   imageUrl?: string
   ethPrice?: string
+  usdPrice?: string
```

Comment:
don't see the serialization code for this line

Reply:
Good catch!